pkg/deployments/action-ids/mainnet/action-ids.json

```diff
@@ -1196,6 +1196,7 @@
     },
     "ERC4626LinearPool": {
       "useAdaptor": false,
+      "factoryOutput": "0x9516a2d25958EdB8da246a320f2c7d94A0DBe25d",
```

Comment:
The contract creator for the pool is not the factory: https://etherscan.io/address/0x9516a2d25958EdB8da246a320f2c7d94A0DBe25d

Maybe that's why there's no `factoryOutput`?

What's the exact problem caused by this?

Reply:
It causes the check-addresses task to fail